hr_planning_resources/models/hr_task_recurrency.py

```diff
@@ -0,0 +1,310 @@
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.date_utils import get_timedelta
+
+TASK_GENERATION_INTERVAL = 1
+_logger = logging.getLogger(__name__)
+
+
+class HrTaskRecurrency(models.Model):
+    _name = "hr.task.recurrency"
+    _description = "Hr Task Recurrency"
+
+    task_ids = fields.One2many(
```

Comment:
```suggestion
    hr_task_ids = fields.One2many(
```
As I said before, prefixing it